tests/msc2716_test.go

```diff
@@ -255,19 +255,87 @@ func TestImportHistoricalMessages(t *testing.T) {
 			})
 		})
 
+		t.Run("Backfill still works after many batches are imported", func(t *testing.T) {
```

Comment:
I don't think 2 batches is "many". The batch itself has many events though, so perhaps rename this test?

Reply:
It will be 11 when we ship, see https://github.com/matrix-org/complement/pull/214#discussion_r747128460